src/request/components/MasterDetailTriPanel.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+interface IMasterDetailTriPanelProps {
+    masterPanel;
+    leftDetailPanel;
+    leftDetailPanelTitle: string;
+    rightDetailPanel;
+    rightDetailPanelTitle: string;
+}
```

Comment:
`Tri` was just the first thing I thought of.  To me, "split" would refer to the split between master/detail panels, not that the detail panel itself is further split into two, hence wanting to emphasize the idea that there were three panels.